src/components/tooltip/tooltip.component.ts

```diff
@@ -181,17 +180,20 @@ export default class SlTooltip extends ShoelaceElement {
 
       // Show
       this.emit('sl-show');
+      document.addEventListener('keydown', this.handleDocumentKeyDown);
```

Comment:
If we bind to the document, we may want to also add a `removeEventListener` in the disconnectedCallback, perhaps an edge case, but generally just a good idea to cleanup

Reply:
Fair point. Added in 6e335f9.